HIPs/incubation/web3j.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: default
+title: Web3j
+parent: Hyperledger Improvement Proposals
+---
+
+# HIP Identifier
+
+Hyperledger Web3j v1.0
+
+# Sponsor(s)
+
+- Conor Svensson [<conor@web3labs.com>](mailto:conor@web3labs.com)
+- Christian Felde [<christian@web3labs.com>](mailto:christian@web3labs.com)
+- George Ţebrean [<george@web3labs.com>](mailto:george@web3labs.com)
+- Nischal Sharma [<nischal@web3labs.com>](mailto:nischal@web3labs.com)
+
+# Abstract
+
+Web3j is the Ethereum integration library for the Java and Android platforms.
+
+# Context
+
+Web3j was first created back in 2016 out of the need to simplify working with Ethereum networks using the Java and Android platforms (collectively referred to as the JVM from herein).
+
+![Web3j provides the plumbing for developing on Ethereum]( https://github.com/web3j/web3j-docs/blob/master/docs/img/web3j_network.png?raw=true)
+
+As someone who had spent more then a decade of their career working on large institutional financial systems on the JVM, I appreciated how much of our modern enterprise infrastructure was built on this platform. 
```

Comment:
Who is "I" in this context? Is this you @conor10 ?

Reply:
It is — should this be articulated differently?

Reply:
While I would like to say yes, I am not sure that I have an alternative to suggest. The thing that I am concerned about is when/if we merge this PR, it might not be clear who the "I" is referring to in the proposal since there is no author provided after the PR.